app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    command = command.split()
+    if command[0] != "cp":
+        raise ValueError("You line should start with cp")
+    if len(command) != 3:
+        raise ValueError("Message should be: cp <source_file> <destination_file>")
+    if command[1] == command[2]:
+        raise ValueError("There should be two different names of files")
+    elif ".txt" in (command[1] and command[2]):
+        with (open(command[1], "r") as file_in,
+              open(command[2], "w") as file_out):
+            file_out.write(file_in.read())
```

Comment:
The use of the 'with' statement is not correct. The 'with' statement should only take one context manager at a time. You need to use nested 'with' statements to open multiple files. Also, you should handle the case where the source file does not exist. Currently, the program will crash if the source file does not exist.